<launch>
  <include file="$(find dji_sdk)/launch/sdk.launch"/>
  <node name="raspicam" pkg="raspicam_node" type="raspicam_node"/>
  <include file="./tag_tracker.launch"/>
  <node name="uav_controller" type="controller_node">
    <param name="arTagSafetyDistance" value=".5"/>
    <param name="maxSpeed" value=".3"/>
  </node>
</launch>
